citadel/install_windows_src.md

```diff
@@ -0,0 +1,136 @@
+# Source Installation on Windows 10
+
+Currently, `ign-gazebo` and `ign-launch` are not supported on Windows 10
+
+Additionally, command line tools as well as the DART physics engine are not currently supported in Windows.
+
+## Install dependencies
+
+1. Install a [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).  Miniconda suffices.
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/). The Community version is free for students, open-source developers, and personal development. Check "Desktop development with C++" in the Workloads tab, and uncheck "C++ Cmake Tools." We will install cmake via Conda.
+1. Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the Windows search field near the Windows button). Optionally, right-click and pin to the task bar for quick access in the future.
+
+    If you did not add Conda to your `PATH` environment variable during Conda installation, you may need to navigate to the location of `condabin` in order to use the `conda` command.
+    To find `condabin`, search for "Anaconda Prompt" in the Windows search field near the Windows button, open it, run `where conda`, and look for a line containing the directory `condabin`.
+
+1. Navigate to your `condabin`, if necessary, and then create and activate a Conda environment:
+
+        conda create -n ign-ws
```

Comment:
Should maintain consistency of code blocks throughout the docs, either indent or using the triple ticks, but not both. They render differently for the per-lib tutorials that use doxygen, so I went with the triple ticks (they look nicer than the indents, wider blocks).

Reply:
72f9eba4f94965dd2bfc9b3a1b778344953a5594